test/api_integration/apis/saved_objects_management/find.ts

```diff
@@ -194,14 +193,18 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     describe('meta attributes injected properly', () => {
-      before(() =>
-        esArchiver.load('test/api_integration/fixtures/es_archiver/management/saved_objects/search')
-      );
-      after(() =>
-        esArchiver.unload(
-          'test/api_integration/fixtures/es_archiver/management/saved_objects/search'
-        )
-      );
+      before(async () => {
+        await kibanaServer.savedObjects.cleanStandardList();
+        await kibanaServer.importExport.load(
+          'test/api_integration/fixtures/kbn_archiver/saved_objects/search.json'
+        );
+      });
+      after(async () => {
+        await kibanaServer.importExport.unload(
```

Comment:
You really don't need this `unload` because the `cleanStandardList` call should delete all the saved objects that you could have loaded (plus any that this test could have created).  But of course it also doesn't hurt.